src/tls13.c

```diff
@@ -8404,6 +8404,75 @@ static word32 NextCert(byte* data, word32 length, word32* idx)
     return len;
 }
 
+#if defined(HAVE_CERTIFICATE_STATUS_REQUEST)
```

Comment:
Fairly certain you can wrap this in `!defined(NO_WOLFSSL_SERVER)` to reduce code size for client only.

Reply:
Thanks